app/models/concept.ts

```diff
@@ -30,6 +36,25 @@ export default class ConceptModel extends Model {
   @equal('status', 'draft') declare statusIsDraft: boolean;
   @equal('status', 'published') declare statusIsPublished: boolean;
 
+  @cached
+  get allBlockGroups(): BlockGroup[] {
```

Comment:
```suggestion
  get blockGroups(): BlockGroup[] {
```

(We don't need "all" here since there's no concept of all/completed - made sense within concept engagement but not here)